test/openAPI/features/voucher_status_check.feature

```diff
@@ -0,0 +1,97 @@
+@method=GET @endpoint=/vouchers/voucherstatuscheck/{voucherserialnumber}
+Feature: The VoucherStatus API is used to check the status of a voucher
+  Check the status of the Voucher. The API will respond with Not Preactivated, Preactivated, Activated, Active, Consumed, Blocked, Suspended and Purged.
+
+  @smoke @positive
+  Scenario: Successfully checks the status of a voucher smoke type test
+    Given Wants to check the status of a voucher
+    When GET request to check the status of the voucher with voucherserialnumer = "5555" is sent
```

Comment:
Is this voucherserialnumer exists in the test-data.json file?
(the same questions below)

Reply:
I will add it, thanks